### DISCUSSION TITLE
Get display text from variable formatting

### DISCUSSION THREAD

**Discussion by thewh1teagle** [grafana/grafana-infinity-datasource#365] [Q&A] [answered]
Hi, thanks for great plugin!
I created grafana variable with two columns - the first act as display text and the second column act as value
I want to get this variable display text and display it on `text` panel
when I use the variable it's display it's value

**Comment by yesoreyeram** [comment, marked as answer]
Looks like a question to grafana text panel. Infinity have nothing to do here.
